packages/core/crates/core-crypto/src/checksum.rs

```diff
@@ -0,0 +1,90 @@
+use sha3::{digest::DynDigest, Keccak256};
+use utils_types::primitives::Address;
+
+// https://eips.ethereum.org/EIPS/eip-55
+pub fn to_checksum(address: Address) -> String {
```

Comment:
Why not making this free function a method of `Address` ?

Reply:
That was my initial thought, but it would need to use `sha3::Keccak256`, which, from what I observed, are separated to `core-crypto` crate. However, in `core-crypto` it's not possible to impl for a "foreign struct"

Reply:
Implement as per suggestion